app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -69,4 +104,15 @@
     white-space: nowrap;
     overflow: scroll;
   }
-}
\ No newline at end of file
+
+  @media screen and (max-width: 1120px) {
+    flex-direction: column;
+    margin-left: 0;
```

Comment:
For consistency, you should zero out both the left and the right margin. Otherwise, you end up with asymmetry like this:

<img width="554" alt="Screen Shot 2020-07-14 at 12 06 04 AM" src="https://user-images.githubusercontent.com/1002748/87395206-da1e2980-c565-11ea-871d-387b135beac6.png">

Even though that orange margin is invisible, it still affects the page layout, and it could have unexpected effects if we make other changes to the layout of the page.